components/omega/doc/devGuide/VerticalMixingCoeff.md

```diff
@@ -0,0 +1,96 @@
+(omega-dev-vertmix) =
+
+# Vertical Mixing Coefficients
+
+Omega includes a `VertMix` class that provides functions that compute `VertDiff` and `VertVisc`, the
+vertical diffusivity and viscosity, where both are defined at the top of cell centers. Currently the
+values of `VertDiff` and `VertVisc` are calculated using the combination of three options: (1) a
+constant background mixing value, (2) a convective instability mixing value, and (3) a Richardson
+number dependent shear mixing value based upon Pacanowski and Philander (1981). These options are
+additive. For both the convective and shear mixing values `BruntVaisalaFreq` is needed, which
+is calculated by the `EOS` class. Currently, the `VertMix` class is set up to have two enumerations,
+`PP` and `KPP`, where the first only uses the three options listed above and the second uses the K Profile Parameterization [(KPP; Large et al., 1994)](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/94rg01872). The `KPP` option is not yet functional (it will throw an error if used), but the framework for its development have been added here.
```

Comment:
this may be out of scope here, but will KPP also be additive?  We likely want cases with KPP in the OSBL, conv instability and background below.  The text suggests you only get all three with PP as the option